scss/modules/_image-focus.scss

```diff
@@ -26,6 +20,12 @@
 
 		width: 100%;
 		height: 100%;
+
+		transition: 800ms all;
```

Comment:
curves gebruiken?

 ik vind eigenlijk dat de notatie in secondes moet. .8s in dit geval, maar ik ben benieuwd of daar misschien artikelen over te vinden zijn welke notatie het beste is.